### DISCUSSION TITLE
Error: Command "npm install" exited with 1

### DISCUSSION THREAD

**Discussion by 0xA15N** [vercel/community#3733] [Help]
### Summary

Help me how to solve this errors

`npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: react-reveal@1.2.2
npm ERR! Found: react@18.2.0
npm ERR! node_modules/react
npm ERR!   react@"18.2.0" from the root project
npm ERR!   peer react@">=18" from @chakra-ui/accordion@2.1.9
npm ERR!   node_modules/@chakra-ui/accordion
npm ERR!     @chakra-ui/accordion@"2.1.9" from @chakra-ui/react@2.5.1
npm ERR!     node_modules/@chakra-ui/react
npm ERR!       @chakra-ui/react@"^2.5.1" from the root project
npm ERR!       1 more (pill-pity)
npm ERR!   108 more (@chakra-ui/alert, @chakra-ui/avatar, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^15.3.0 || ^16.0.0" from react-reveal@1.2.2
npm ERR! node_modules/react-reveal
npm ERR!   react-reveal@"^1.2.2" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: react@16.14.0
npm ERR! node_modules/react
npm ERR!   peer react@"^15.3.0 || ^16.0.0" from react-reveal@1.2.2
npm ERR!   node_modules/react-reveal
npm ERR!     react-reveal@"^1.2.2" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! /vercel/.npm/_logs/2023-08-22T21_24_46_847Z-eresolve-report.txt
npm ERR! A complete log of this run can be found in:
npm ERR!     /vercel/.npm/_logs/2023-08-22T21_24_46_847Z-debug-0.log
Error: Command "npm install" exited with 1`

### Example

https://github.com/0xA15N/zecca

### Steps to Reproduce

i got this code while deploying:

`npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: react-reveal@1.2.2
npm ERR! Found: react@18.2.0
npm ERR! node_modules/react
npm ERR!   react@"18.2.0" from the root project
npm ERR!   peer react@">=18" from @chakra-ui/accordion@2.1.9
npm ERR!   node_modules/@chakra-ui/accordion
npm ERR!     @chakra-ui/accordion@"2.1.9" from @chakra-ui/react@2.5.1
npm ERR!     node_modules/@chakra-ui/react
npm ERR!       @chakra-ui/react@"^2.5.1" from the root project
npm ERR!       1 more (pill-pity)
npm ERR!   108 more (@chakra-ui/alert, @chakra-ui/avatar, ...)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^15.3.0 || ^16.0.0" from react-reveal@1.2.2
npm ERR! node_modules/react-reveal
npm ERR!   react-reveal@"^1.2.2" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: react@16.14.0
npm ERR! node_modules/react
npm ERR!   peer react@"^15.3.0 || ^16.0.0" from react-reveal@1.2.2
npm ERR!   node_modules/react-reveal
npm ERR!     react-reveal@"^1.2.2" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! /vercel/.npm/_logs/2023-08-22T21_24_46_847Z-eresolve-report.txt
npm ERR! A complete log of this run can be found in:
npm ERR!     /vercel/.npm/_logs/2023-08-22T21_24_46_847Z-debug-0.log
Error: Command "npm install" exited with 1`

Screenshot:
![Screenshot_20230823-184103_Chrome](https://github.com/vercel/community/assets/75478154/ae197be0-9167-4ee9-9954-067bfe755104)

Link: [Repository](https://github.com/0xA15N/zecca)

**Comment by shadoath** [comment]
Try adding an environmental value of
‘VERCEL_CLI_VERSION’ to ‘vercel@31.1.1’
Or the latest version that worked for you. You can find it in the build logs. 
![IMG_2818](https://github.com/vercel/community/assets/1946020/54970780-fe09-4dc1-ae2d-941a5feaeba7)

**Comment by dpaulos6** [comment]
Mine shows this error and I'm unable to find a fix for this, may someone please help me?

```
[16:08:38.804] npm ERR! code 1
[16:08:38.806] npm ERR! path /vercel/path0/node_modules/@fortawesome/fontawesome-common-types
[16:08:38.806] npm ERR! command failed
[16:08:38.807] npm ERR! command sh -c node attribution.js
[16:08:38.807] npm ERR! node: /lib64/libm.so.6: version `GLIBC_2.27' not found (required by node)
[16:08:38.807] npm ERR! node: /lib64/libc.so.6: version `GLIBC_2.27' not found (required by node)
[16:08:38.807] npm ERR! node: /lib64/libc.so.6: version `GLIBC_2.28' not found (required by node)
[16:08:38.808] 
[16:08:38.809] npm ERR! A complete log of this run can be found in: /vercel/.npm/_logs/2023-12-15T16_08_20_656Z-debug-0.log
[16:08:38.846] Error: Command "npm install" exited with 1
```

I've tried adding the env var as replied before, and it still didn't work.
I'm currently in a timed race to finish this project and I can't build it, so I'm very frustrated.

**Please Help Me!**
